src/server/run_sched.c

```diff
@@ -359,9 +365,9 @@ schedule_high(pbs_sched *psched)
 		psched->svr_do_sched_high = SCH_SCHEDULE_NULL;
 		return 0;
 	}
-	strncpy(psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str, SC_SCHEDULING, SC_STATUS_LEN);
-	psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str[SC_STATUS_LEN] = '\0';
-	psched->sch_attr[(int) SCHED_ATR_sched_state].at_flags = ATR_VFLAG_DEFLT | ATR_VFLAG_SET | ATR_VFLAG_MODCACHE;
```

Comment:
You may still want to retain this line to add "ATR_VFLAG_DEFLT"  (in this place and all other places) because no setter function will add this for you.

Reply:
It looks we are not really making use of this attribute in Multisched code and hence not using the flag.